src/languages/en.js

```diff
@@ -168,4 +168,115 @@ export default {
+    secureOptionTLS: "TLS (465)",
+    "Ignore TLS Error": "Ignore TLS Error",
+    "From Email": "From Email",
+    "To Email": "To Email",
+    smtpCC: "CC",
+    smtpBCC: "BCC",
+
+    "discord": "Discord",
+    "Discord Webhook URL": "Discord Webhook URL",
+    wayToGetDiscordURL: "You can get this by going to Server Settings -> Integrations -> Create Webhook",
+    "Bot Display Name": "Bot Display Name",
+    "Prefix Custom Message": "Prefix Custom Message",
+    "Hello @everyone is...": "Hello @everyone is...",
+
+    "teams": "Microsoft Teams",
+    "Webhook URL": "Webhook URL",
+    wayToGetTeamsURL: "You can learn how to create a webhook url {0}.",
+
+    "signal": "Signal",
+    "Number": "Number",
+    "Recipients": "Recipients",
+    needSignalAPI: "You need to have a signal client with REST API.",
+    wayToCheckSignalURL: "You can check this url to view how to setup one:",
+    signalImportant: "IMPORTANT: You cannot mix groups and numbers in recipients!",
+
+    "gotify": "Gotify",
+    "Application Token": "Application Token",
+    "Server URL": "Server URL",
+    "Priority": "Priority",
+
+    "slack": "Slack",
+    "Icon Emoji": "Icon Emoji",
+    "Channel Name": "Channel Name",
+    "Uptime Kuma URL": "Uptime Kuma URL",
+    aboutWebhooks: "More info about webhooks on: {0}",
+    aboutChannelName: "Enter the channel name on {0} Channel Name field if you want to bypass the webhook channel. Ex: #other-channel",
+    aboutKumaURL: "If you leave the Uptime Kuma URL field blank, it will default to the Project Github page.",
+    emojiCheatSheet: "Emoji cheat sheet: {0}",
+
+    "rocket.chat": "Rocket.chat",
+
+    "pushover": "Pushover",
+    "User Key": "User Key",
+    "Device": "Device",
+    "Message Title": "Message Title",
+    "Notification Sound": "Notification Sound",
+    "More info on:": "More info on: {0}",
+    pushoverDesc1: "Emergency priority (2) has default 30 second timeout between retries and will expire after 1 hour.",
+    pushoverDesc2: "If you want to send notifications to different devices, fill out Device field.",
+
+    "pushy": "Pushy",
+
+    "octopush": "Octopush",
+    "SMS Type": "SMS Type",
+    octopushTypePremium: "Premium (Fast - recommended for alerting)",
+    octopushTypeLowCost: "Low Cost (Slow, sometimes blocked by operator)",
+    "Check octopush prices": "Check octopush prices {0}.",
+    octopushPhoneNumber: "Phone number (intl format, eg : +33612345678) ",
+    octopushSMSSender: "SMS Sender Name : 3-11 alphanumeric characters and space (a-zA-Z0-9)",
+
+    "lunasea": "LunaSea",
+    "LunaSea Device ID": "LunaSea Device ID",
+
+    "apprise": "Apprise (Support 50+ Notification services)",
+    "Apprise URL": "Apprise URL",
+    "Example:": "Example: {0}",
+    "Read more:": "Read more: {0}",
+    "Status:": "Status: {0}",
+    "Read more": "Read more",
+    "Apprise is installed.": "Apprise is installed. ",
+    "Apprise is not installed.": "Apprise is not installed. ",
```

Comment:
```suggestion
    "Apprise is installed.": "Apprise is installed.",
    "Apprise is not installed.": "Apprise is not installed.",
```

Reply:
Some languages use Full-width symbols, such as the period "。" in Chinese/Japanese, which don't need space after a period.
I know it's probably redundant, but I want to ensure it's unnecessary most time?

Reply:
If some languages use that - language translators will add them. For base language (English) space is redundant, because we have space before `<a href>` tag (see my previous suggestion

Reply:
I mean the space is redundant for other languages, not redundant for English.

> Apprise is not installed. [Read more]()
> Apprise 没有安装。[了解更多]()

No space after "。"

Reply:
Maybe I should add a slot?

> "Apprise is not installed. {0}"

Reply:
I have not checkout yet, but it should not be a problem, since there is no follow words.

![image](https://user-images.githubusercontent.com/1336778/134365228-06b74e28-a581-4ebe-9515-5fda8f17d3f1.png)

Reply:
> I have not checkout yet, but it should not be a problem, since there is no follow words.
> 
> ![image](https://user-images.githubusercontent.com/1336778/134365228-06b74e28-a581-4ebe-9515-5fda8f17d3f1.png)

"Read more" only appear after is not installed